### DISCUSSION TITLE
When will codewars update Golang version?

### DISCUSSION THREAD

**Discussion by illiafox** [codewars/codewars.com#2634] [Q&A] [answered]
HI! Today golang devs released [go 1.18](https://github.com/golang/go/milestone/201)

Although, codewars still uses [go 1.12](https://go.dev/doc/go1.12), which was released 3 years ago!
![Screenshot from 2022-03-15 20-00-04](https://user-images.githubusercontent.com/61962654/158442291-2f484486-fbed-4bf1-a5e8-eda1fc33de8c.png)

What's the problem with updating to newer version?

**Comment by kazk** [comment, marked as answer]
You can request a new version by opening an issue with the template in https://github.com/codewars/runner

**Reply by twocs** [reply]
I also vote for a new version.
@illiafox opened this issue on 23 Mar, 2022: https://github.com/codewars/runner/issues/176
And looks like it's "In Progress", which is good to see.
